oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml

```diff
@@ -121,3 +121,13 @@ feishuHooks:
       secret: dummysecret
     - url: https://open.feishu.cn/open-apis/bot/v2/hook/dummy_token2
       secret:
+
+weLinkHooks:
+  textTemplate: "Apache SkyWalking Alarm: \n %s."
+  webhooks:
+    - client_id: "19950212172337588234567"
```

Comment:
We need to be clear about these ID/secret(s) are dummy, need to change.

Reply:
I can add some comment above this line and doc.

Reply:
My question is, are these IDs real? If so, this is a security leak. If not, please put other words on this.